system-helper/org.freedesktop.Flatpak.rules.in

```diff
@@ -7,5 +7,13 @@ polkit.addRule(function(action, subject) {
         subject.active == true && subject.local == true &&
         subject.isInGroup("@privileged_group@")) {
             return polkit.Result.YES;
+    } else if ((action.id == "org.freedesktop.Flatpak.app-install" ||
+                action.id == "org.freedesktop.Flatpak.install-bundle") &&
```

Comment:
I am suspicious of this install-bundle being here. Doesn't this mean the user can introduce a bundle file which contains unbounded profanity, with an inaccurate appdata file, which otherwise circumvents the trust that we place in the appdata which comes from a repo that the admin/distro has vetted/trusted/...? Basically, unpriveleged users should not be able to create or modify repository definitions because these are the trust anchors from which we get our ratings (and indeed the whole confidence of system administrators that unpriveleged users installing stuff system-wide might ever be desirable). If installing a bundle works outside of these trust anchors, we can't grant it.

Reply:
I thought bundles had to contain signed commits, but I know so little about them (and we don’t use them), that it’s probably safer to drop this bit of the commit.

Reply:
If `--gpg-keys` isn't used with `flatpak build-bundle` when the bundle is created, GPG verification won't be used when installing it.

Reply:
I’ve dropped the bundle change from the `.rules` file.